templates/Layout/DNRoot_environment.ss

```diff
@@ -37,49 +18,66 @@
 					$DeploymentMessages
 				<% end_if %>
 
-				<h4>Environment Details:</h4>
-
-				<ul>
+				<h3>Currently deployed  
 					<%-- Display Environment URL --%>
 					<% if $URL %>
-						<li>
-							<span class="fa fa-link"></span>
-							<a href="$URL">View site at $BareURL</a>
-						</li>
+						<small><a class="text-muted" href="$URL"><% if $BareURL %>$BareURL<% else %>$URL<% end_if %></a></small>
```

Comment:
Did you have a situation where BareURL wasn't returning anything, but URL was set?